test/api/ApolloGraphClient.test.ts

```diff
@@ -16,7 +16,7 @@ import {
 
 describe("ApolloGraphClient", () => {
 
-    describe("api", () => {
+    describe.skip("api", () => {
```

Comment:
?

Reply:
Skip Apollo API tests
since GitHub token auth is no longer supported.